src/components/label-pill/LabelPill.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import Tooltip, { TooltipPosition } from '../tooltip';
+
+import './LabelPill.scss';
+
+export enum StatusLevelType {
```

Comment:
I would probably prefix this with `LabelPill` so that we don't have ambiguous exports. Also is the word `type` needed? Can it just be statusLevel (even the prop name)? Or maybe its just `type` and no status level